pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -239,7 +239,7 @@ class RefactoringChecker(checkers.BaseTokenChecker):
             "with statement assignment and exception handler assignment.",
         ),
         "R1705": (
-            'Unnecessary "%s" after "return"',
+            'Unnecessary "%s" after "return" remove it and de-indent all the code inside it',
```

Comment:
I think here should be a comma before remove, in all the cases

Reply:
https://github.com/PyCQA/pylint/pull/5736/commits/a281fadc650072b7737f3b0bb6485011c5f78db4